packages/@webex/plugin-meetings/src/statsAnalyzer/index.ts

```diff
@@ -1094,73 +1094,35 @@ export class StatsAnalyzer extends EventsScope {
     if (!result || !result.id) {
       return;
     }
-    const RemoteCandidateType = {};
-    const RemoteTransport = {};
-    const RemoteIpAddress = {};
-    const RemoteNetworkType = {};
 
-    if (!result.id) return;
+    let transport;
+    if (result.relayProtocol) {
+      transport = result.relayProtocol.toUpperCase();
+    } else if (result.protocol) {
+      transport = result.protocol.toUpperCase();
+    }
 
     const sendRecvType = isSender ? STATS.SEND_DIRECTION : STATS.RECEIVE_DIRECTION;
     const ipType = isRemote ? STATS.REMOTE : STATS.LOCAL;
 
-    if (!RemoteCandidateType[result.id]) {
-      RemoteCandidateType[result.id] = [];
-    }
-
-    if (!RemoteTransport[result.id]) {
-      RemoteTransport[result.id] = [];
-    }
-
-    if (!RemoteIpAddress[result.id]) {
-      RemoteIpAddress[result.id] = [];
-    }
-    if (!RemoteNetworkType[result.id]) {
-      RemoteNetworkType[result.id] = [];
-    }
-
-    if (
-      result.candidateType &&
-      RemoteCandidateType[result.id].indexOf(result.candidateType) === -1
-    ) {
-      RemoteCandidateType[result.id].push(result.candidateType);
-    }
-
-    if (result.protocol && RemoteTransport[result.id].indexOf(result.protocol) === -1) {
-      RemoteTransport[result.id].push(result.protocol.toUpperCase());
-    }
-
-    if (
-      result.ip &&
-      RemoteIpAddress[result.id].indexOf(`${result.ip}:${result.portNumber}`) === -1
-    ) {
-      RemoteIpAddress[result.id].push(`${result.ip}`); // TODO: Add ports
-    }
-
-    if (result.networkType && RemoteNetworkType[result.id].indexOf(result.networkType) === -1) {
-      RemoteNetworkType[result.id].push(result.networkType);
-    }
-
     this.statsResults.internal.candidates[result.id] = {
-      candidateType: RemoteCandidateType[result.id],
-      ipAddress: RemoteIpAddress[result.id],
+      candidateType: result.candidateType,
+      ipAddress: result.ip, // TODO: add ports
       portNumber: result.port,
-      networkType: RemoteNetworkType[result.id],
+      networkType: result.networkType,
       priority: result.priority,
-      transport: RemoteTransport[result.id],
+      transport,
       timestamp: result.time,
       id: result.id,
       type: result.type,
     };
 
-    this.statsResults.connectionType[ipType].candidateType = RemoteCandidateType[result.id];
-    this.statsResults.connectionType[ipType].ipAddress = RemoteIpAddress[result.id];
+    this.statsResults.connectionType[ipType].candidateType = result.candidateType;
+    this.statsResults.connectionType[ipType].ipAddress = result.ipAddress;
 
     this.statsResults.connectionType[ipType].networkType =
-      RemoteNetworkType[result.id][0] === NETWORK_TYPE.VPN
-        ? NETWORK_TYPE.UNKNOWN
-        : RemoteNetworkType[result.id][0];
-    this.statsResults.connectionType[ipType].transport = RemoteTransport[result.id];
+      result.networkType === NETWORK_TYPE.VPN ? NETWORK_TYPE.UNKNOWN : result.networkType;
```

Comment:
can you find out why we set networkType to UNKNOWN if browser says it's VPN? it feels like knowing that it's VPN would be a useful piece of information, so I wonder why we ignore that.

@arun3528  maybe you know?